support-frontend/yarn.lock

```diff
@@ -1661,6 +1661,34 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@floating-ui/core@^1.2.1":
```

Comment:
Quite a few new floating dependencies

Reply:
> Sorry Maria, took me quite a while to grasp how works. Works great so have signed off. Might be worth asking (experienced) Imogen to double check too if you wish? Great job!

I'd say it'd be great if you wouldn't mind checking this out on CODE and testing out the requirements above. If you had already done so I'd say this should be good to go in!